sormas-backend/src/main/java/de/symeda/sormas/backend/event/EventParticipantFacadeEjb.java

```diff
@@ -513,12 +509,66 @@ public void delete(String uuid, DeletionDetails deletionDetails) throws External
 		service.delete(eventParticipant, deletionDetails);
 	}
 
+	@Override
+	@RightsAllowed(UserRight._EVENTPARTICIPANT_DELETE)
+	public void delete(List<String> uuids, DeletionDetails deletionDetails) {
+		deleteEventParticipants(uuids, deletionDetails);
+	}
+
+	@Override
+	@RightsAllowed(UserRight._EVENTPARTICIPANT_DELETE)
+	public List<String> deleteEventParticipants(List<String> uuids, DeletionDetails deletionDetails) {
```

Comment:
Probably makes sense to also add a default implementation for the delete method because we have the same implementation at least for some entities here as well, i.e. this would reduce code duplication

Reply:
Same as above